src/test/java/org/takes/facets/fallback/TkFallbackTest.java

```diff
@@ -104,4 +104,23 @@ public InputStream body() {
         );
     }
 
+    /**
+     * TkFallback can fall back.
+     * @throws IOException If some problem happens
+     */
+    @Test
+    public void fallsBackWithProperMessage() throws IOException {
+        try {
+            new TkFallback(
+                new TkFailure(),
+                new FbChain()
+              ).act(new RqFake());
```

Comment:
@danff indentation seems wrong here. let's align it with the `new` keyword on line 114